framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/KieServerInstanceImpl.java

```diff
@@ -54,6 +65,22 @@ public void setPodName(String podName) {
         return podName;
     }
 
+    @Override public CommandExecutionResult runCommand(String... command) {
```

Comment:
I see that, runCommand method is same for all instances. Can we add implementation into some parent class e.g. OpenShiftInstance to avoid duplicate code?